src/ResourceManager/Sql/Commands.Sql/help/AzureRM.Sql.md

```diff
@@ -203,6 +203,9 @@ This command creates a new Azure SQL Database Failover Group.
 ### [New-AzureRmSqlDatabaseImport](New-AzureRmSqlDatabaseImport.md)
 Imports a .bacpac file and create a new database on the server.
 
+### [New-AzureRmSqlDatabaseRestorePoint](New-AzureRmSqlDatabaseRestorePoint.md)
```

Comment:
@jugeorge this module page will need to be regenerated since it does not include all of the added cmdlets

Reply:
the 5 added cmdlets seem to be there now.